examples/chat/home.html

```diff
@@ -36,7 +36,7 @@
             appendLog(item);
         };
         conn.onmessage = function (evt) {
-            var messages = evt.data.split('\n');
+            var messages = JSON.parse(evt.data).body.split('\n');
```

Comment:
Use JSON array for multiple chat messages instead of stuffing multiple messages in message.body.

    var messages = JSON.parse(evt.Data)
    ...